types/source-map-support/index.d.ts

```diff
@@ -1,16 +1,24 @@
 // Type definitions for source-map-support 0.5
```

Comment:
Should this be 0.6 since it depends on `"source-map": "^0.6.0"` ?

Reply:
`source-map-support "0.5.19"` requires `source-map "^0.6.0"`
https://github.com/evanw/node-source-map-support/blob/master/package.json#L14